Sources/Clappr_iOS/Classes/Plugin/Base/JumpPlugin.swift

```diff
@@ -1,22 +1,22 @@
 import UIKit
 
-public class DoubleTapPlugin: UICorePlugin {
+public class JumpPlugin: UICorePlugin {
     
-    var doubleTapGesture: UITapGestureRecognizer!
```

Comment:
Can we keep the gesture name as doubleTap? Just to keep the name reflecting what gesture it refers to.

Reply:
Done.